### PR TITLE
Add explicit model attribute to make filtering by model easier.

### PR DESCRIPTION
**Pull request by schnecle**
Checklist (if applicable):
- [x] Tested (manually, unit tested, etc.)
- [ ] Docs updated

### CODE REVIEW

File: js/plugins/google-cloud/tests/traces_test.ts [hunk 1]
```diff
@@ -14,10 +14,13 @@
  * limitations under the License.
  */
 
+import { runWithRegistry } from '@genkit-ai/core/registry';
 import { ReadableSpan } from '@opentelemetry/sdk-trace-base';
 import { Genkit, genkit, run, z } from 'genkit';
 import assert from 'node:assert';
 import { after, before, beforeEach, describe, it } from 'node:test';
+import { generate } from '../../../ai/src/generate.js';
```
Comment (schnecle) [inline js/plugins/google-cloud/tests/traces_test.ts:22]: Should these be different imports?

Comment (maxl0rd) [reply js/plugins/google-cloud/tests/traces_test.ts:22]: Yes, they should probably be `from '@genkit-ai/ai`

File: js/plugins/google-cloud/src/gcpOpenTelemetry.ts [hunk 2]
```diff
@@ -338,6 +339,13 @@ class AdjustingTraceExporter implements SpanExporter {
     }
     return span;
   }
+
+  private markGenkitModel(span: ReadableSpan): ReadableSpan {
+    if (span.attributes['genkit:metadata:subtype'] === 'model') {
```
Comment (maxl0rd) [inline js/plugins/google-cloud/src/gcpOpenTelemetry.ts:344]: In the other examples, I added `&& attributes['genkit:name']` to the condition, just to avoid the corner case where we write a null key. Shouldn't ever happen but...

File: js/plugins/google-cloud/tests/traces_test.ts [hunk 3]
```diff
@@ -132,6 +135,50 @@ describe('GoogleCloudTracing', () => {
     assert.equal(spans[1].attributes['genkit/feature'], 'niceFlow');
   });
 
+  it('adds the genkit/model label for model actions', async () => {
+    const echoModel = runWithRegistry(ai.registry, () =>
```
Comment (maxl0rd) [inline js/plugins/google-cloud/tests/traces_test.ts:139]: runWithRegistry doesn't return anything

**Review by maxl0rd — COMMENTED** [review]
Just a few nits

File: js/plugins/google-cloud/tests/traces_test.ts (hunk 3 above)
Comment (schnecle) [reply js/plugins/google-cloud/tests/traces_test.ts:139]: then how is this passing?!

File: js/plugins/google-cloud/src/gcpOpenTelemetry.ts (hunk 2 above)
Comment (schnecle) [reply js/plugins/google-cloud/src/gcpOpenTelemetry.ts:344]: Good call!

File: js/plugins/google-cloud/tests/traces_test.ts (hunk 3 above)
Comment (schnecle) [reply js/plugins/google-cloud/tests/traces_test.ts:139]: I think this does some magic because it looks like we are using it to return stuff all over the place. Example: https://github.com/firebase/genkit/blob/next/js/ai/tests/generate/generate_test.ts#L595

File: js/plugins/google-cloud/tests/traces_test.ts (hunk 1 above)
Comment (schnecle) [reply js/plugins/google-cloud/tests/traces_test.ts]: it actually looks like it's now 'genkit' and 'genkit/model'

File: js/plugins/google-cloud/tests/traces_test.ts (hunk 3 above)
Comment (schnecle) [reply js/plugins/google-cloud/tests/traces_test.ts:138]: There is no declared return type, but the function is returning whatever the internal function is doing.

**Review by maxl0rd — APPROVED** [review]
